openshift/scripts/oc_provision_c_haines_cronjob.sh

```diff
@@ -33,10 +33,11 @@ OC_PROCESS="oc -n ${PROJ_TARGET} process -f ${TEMPLATE_PATH}/c_haines.cronjob.ya
 -p IMAGE_NAME=${APP_NAME}-api-${SUFFIX} \
 -p IMAGE_TAG=${SUFFIX} \
 -p SUFFIX=${SUFFIX} \
--p POSTGRES_USER=wps \
--p POSTGRES_DATABASE=wps \
--p POSTGRES_WRITE_HOST=patroni-wps-${SUFFIX}-leader \
--p POSTGRES_READ_HOST=patroni-wps-${SUFFIX}-replica \
+-p POSTGRES_DATABASE=${POSTGRES_DATABASE:-${APP_NAME}} \
+-p POSTGRES_USER=wps-crunchydb-${SUFFIX} \
+-p POSTGRES_WRITE_HOST=wps-crunchydb-${SUFFIX}-primary \
+-p POSTGRES_READ_HOST=wps-crunchydb-${SUFFIX}-primary \
```

Comment:
Maybe a question to answer later, but why switch from having different read/write to the same read/write hosts? Just curiosity

Reply:
I think the original thinking is that all reads should come from replicas to avoid contention on the primary. Right now with storage constraints we just have one primary. Once the old cluster is wound down we can spin up replicas and re-adjust. Ideally we should be pointing the `pgbouncer` host that handles postgres connection load balancing for us.